--- src/chezmoi/.chezmoidata/gemini.toml
+++ src/chezmoi/.chezmoidata/gemini.toml
@@ -1,45 +1,45 @@
 # Configuration documentation: https://geminicli.com/docs/get-started/configuration/

 [gemini]
 model = "auto-gemini-3"

-[gemini.telemetry]
+[telemetry]
 enabled = false
 log_prompts = false

-[gemini.ui]
+[ui]
 show_line_numbers = true
 show_memory_usage = true
 show_model_info_in_chat = true

-[gemini.ui.footer]
+[ui.footer]
 hide_context_percentage = false

-[gemini.general]
+[general]
 enable_auto_update = false
 preview_features = true
 vim_mode = true

 # Session Management: https://developers.googleblog.com/pick-up-exactly-where-you-left-off-with-session-management-in-gemini-cli/
-[gemini.general.sessionRetention]
+[general.sessionRetention]
 enabled = true
 max_age = "30d"
 warning_acknowledged = true

 # Sandbox Configuration: https://geminicli.com/docs/cli/sandbox/
-[gemini.tools]
+[tools]
 sandbox = true

-[gemini.extensions]
+[extensions]

-  [gemini.extensions.conductor]
+  [extensions.conductor]
   url = "https://github.com/gemini-cli-extensions/conductor"
   ref = "conductor-v0.4.1"
   installation = "gemini-cli"

-[gemini.context]
+[context]
 include_directories = ["~/.config/gh"]

-[gemini.experimental]
+[experimental]
 worktrees = true
